Gemfile.d/app.rb

```diff
@@ -163,4 +163,4 @@
 gem 'workflow', path: 'gems/workflow'
 
 # StrongMind
-gem 'canvas_shim', '0.1.12'
+gem 'canvas_shim', git: 'https://github.com/StrongMind/canvas_shim.git'
```

Comment:
I'm concerned that this might lead to unintentional production deploys. Can we have a chat about how this will work?

Reply:
Pushed version 0.1.13